src/main/resources/openapi/dailyList.yaml

```diff
@@ -36,11 +36,11 @@ paths:
         '500':
           description: Internal Server Error
 
-  /dailylist/getCases:
+  /dailylists/getCases:
```

Comment:
I guess we should change our paths here to be `/dailylists/get-cases`?
[Must: Use lowercase separate words with hyphens for Path Segments [129]](https://hmcts.github.io/restful-api-standards/#129)

Reply:
Agreed